### ISSUE TITLE
Add ssh key of gislab user to gislab client 

### ISSUE THREAD

**Issue by lfurtkevicova** [gislab-npo/gislab#487]
Currently it is not possible to log in to the gislab client from the server by ssh.

**Comment by landam** [comment]
Confirmed, currently I am bypassing it by special user after-add hook.

**Comment by imincik** [comment]
I don't understand this issue. Can you explain this issue in detail ?

**Comment by landam** [comment]
You cannot connect from the server to the running client:

```
ssh gislab@c50
```

says 'Error: Permission denied (publickey)'.

**Comment by imincik** [comment]
It is true, only SSH keys authentication is allowed. Do you propose creating some password-less SSH keys on user account creation ?

**Comment by landam** [comment]
Yes, the `gislab` users should definitely be able to access running clients. Something like

```
mkdir /mnt/home/${GISLAB_USER}/.ssh
cp /mnt/home/gislab/.ssh/id_dsa.pub /mnt/home/${GISLAB_USER}/.ssh/authorized_keys
chown ${GISLAB_USER}:gislabusers /mnt/home/${GISLAB_USER}/.ssh/ -R
```

**Comment by imincik** [comment]
'gislab' user account doesn't contain private or public SSH key.

But it contains public key used for provisioning user (ubuntu or vagrant). So adding this key to ssh-agent ($ ssh-add <key>) and logging with agent forwarding enabled ($ ssh -A) should allow logging to server and jumping to clients as well.

**Comment by imincik** [comment]
Run "$ vagrant ssh-config" to get SSH connection information used by Vagrant which contains also path to SSH key used by Vagrant

**Comment by landam** [comment]
I don't understand, there is no vagrant used/installed on GIS.lab unit, right?

**Comment by imincik** [comment]
@landam, I meant that every user account used for GIS.lab installation ('vagrant' if using Vagrant, 'ubuntu' for other installations) contains public SSH key. For GIS.lab Unit, it is the one which is specified during basic OS ISO image creation. This key is copied during installation also to 'gislab' user account (see: /storage/home/gislab/.ssh/authorized_keys). Therefore, you can use this SSH key for connection. See ssh-agent for more information how to forward SSH keys.

**Comment by landam** [comment]
OK, but I still think that gislab user should be able to connect from the server to the clients out of box (without any extra effort). 

```
ssh gislab@c50
```
